automation/taskcluster/decision_task.py

```diff
@@ -131,6 +131,45 @@ def release(channel, is_staging, version_name):
         signing_task_id,
         apks=apk_paths,
         channel=channel,
+        # TODO until org.mozilla.fenix.nightly is made public, put it on the internally-testable track
+        override_google_play_track=None if channel != "nightly" else "internal",
```

Comment:
Note: we want to test `org.mozilla.fenix.nightly` before it's released.
By default, nightlies are published to the `beta` track, which is public.
So, we're going to override and publish on `internal` for a while, so we can internally test. Once `org.mozilla.fenix.nightly` is made public, we'll put this back to normal, so each `nightly` is automatically published publicly